### PR TITLE
Use official rollbar library

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

**Comment by Soulou** [comment]
Do not merge before related PR https://github.com/Scalingo/logrus-rollbar is ready

**Comment by johnsudaar** [comment]
Ok notify us when ready?

**Comment by Soulou** [comment]
it's  already done 1.3.0

File: Gopkg.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 [[constraint]]
   name = "github.com/Scalingo/logrus-rollbar"
-  version = "1.2.1"
+  branch = "deps/new-rollbar"
```
Comment (johnsudaar) [inline Gopkg.toml]: Not here :/

**Pull request merged**